docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1e.md

```diff
@@ -0,0 +1,56 @@
+# does_zone_meet_G3_1_1e
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a given zone meets the G3_1_1e exception "Thermal zones designed with heating-only systems in the proposed design serving storage rooms, stairwells, vestibules, electrical/mechanical rooms, and restrooms not exhausting or transferring air from mechanically cooled thermal zones in the proposed design shall use system type 9 or 10 in the baseline building design."
+
+**Inputs:**
+- **P-RMD**
+- **B-RMD**
+- **zone_id**
+
+**Returns:**  
+- **result**: an ENUM YES, NO, LIKELY_VESTIBULE or MAYBE_VESTIBULE
+ 
+**Function Call:**
+- **is_a_vestibule**
+- **is_zone_mechanically_cooled**
+- **get_zone** - Weili mentioned a get_zone function, but I don't see it in the docs, I'm assuming we can pass an RMD and zone_id and get a zone?
+
+## Logic:
+- create list of eligible space types: `eligible_space_types = ["STORAGE_ROOM_HOSPITAL", "STORAGE_ROOM_SMALL", "STORAGE_ROOM_LARGE", "WAREHOUSE_STORAGE_AREA_MEDIUM_TO_BULKY_PALLETIZED_ITEMS", "WAREHOUSE_STORAGE_AREA_SMALLER_HAND_CARRIED_ITEMS", "STAIRWELL", "ELECTRICAL_MECHANICAL_ROOM", "RESTROOM_FACILITY_FOR_THE_VISUALLY_IMPAIRED", "RESTROOM_ALL_OTHERS"]
+- set the result variable to No - only a positive test can give it a different value: `result = NO`
+- set eligibility boolean to False: `eligible = TRUE`
+- get the zone in the P-RMD: `zone_p = get_zone(P-RMD,zone_id)`
+- get the zone in the B-RMD: `zone_b = get_zone(B-RMD,zone_id)`
+- For each space in zone_b: `for space_p in zone_b.Spaces:`
+	- check if the space has an eligible lighting type: `if space_p.lighting_space_type not in eligible_space_types:`
+		- set eligibility to False - any non-compliant space type will result in a non-eligible zone: `eligible = FALSE`
+
+- if the zone is not eligible, check if it is a vestibule: `if not eligible:`
+	- set variable is_a_vestibule = is_zone_a_vestibule: `is_a_vestibule = is_zone_a_vestibule(zone_b, B-RMD)`
+	- if the result is not equal to no, then it is eligible to be a vestibule: `if is_a_vestibule != NO:`
+		- set eligible to true: `eligible = TRUE`
+
+- if the zone is still eligible: `if eligible:`
+	- check if the proposed has a heating-only system by using the is_zone_mechanically_cooled function: `if is_zone_mechanically_cooled(P-RMD,zone_p)`
+		- mechanically cooled zones are not eligible: `eligible = FALSE`
```

Comment:
Here, we would call the proposed new function "does_zone_gets_cooled_transfer_air", if yes, then `eligible = FALSE`